src/main/java/edu/cornell/kfs/concur/batch/ConcurStandardAccountingExtractToPdpAndCollectorStep.java

```diff
@@ -41,11 +43,23 @@ protected boolean processCurrentFileAndExtractPdpFeedFromSAEFile(String saeFileN
         ConcurStandardAccountingExtractFile concurStandardAccoutingExtractFile = getConcurStandardAccountingExtractService()
                 .parseStandardAccoutingExtractFile(saeFileName);
         if (getConcurStandardAccountingExtractValidationService().validateConcurStandardAccountExtractFile(concurStandardAccoutingExtractFile)) {
-            success = getConcurStandardAccountingExtractService().extractPdpFeedFromStandardAccounitngExtract(concurStandardAccoutingExtractFile);
+            String outputFileName = getConcurStandardAccountingExtractService().extractPdpFeedFromStandardAccounitngExtract(concurStandardAccoutingExtractFile);
+            if (StringUtils.isEmpty(outputFileName)) {
+                success = false;
+                LOG.error("processCurrentFileAndExtractPdpFeedFromSAEFile, could not produce a PDP XML file for " + saeFileName);
+            }
             if (success) {
                 success &= getConcurStandardAccountingExtractService()
                         .extractCollectorFeedFromStandardAccountingExtract(concurStandardAccoutingExtractFile);
             }
+            if (success) {
+                try {
+                    getConcurStandardAccountingExtractService().createDoneFileForPdpFile(outputFileName);
+                } catch (IOException e) {
+                    LOG.error("processCurrentFileAndExtractPdpFeedFromSAEFile, unable to create .done file: ", e);
+                    success = false;
+                }
+            }
```

Comment:
I think we want to leave the PDP xml in place.  There won't be a .done file so it won't process. The file may be helpful in analyzing a problem.  Also, if we need to re-process the same file, the existing pdp xml will get over-written.